src/main/java/se/sundsvall/billingdatacollector/Application.java

```diff
@@ -2,12 +2,24 @@
 
 import static org.springframework.boot.SpringApplication.run;
 
+import org.springframework.context.ConfigurableApplicationContext;
+
 import se.sundsvall.dept44.ServiceApplication;
 
 @ServiceApplication
 public class Application {
 
+	private static ConfigurableApplicationContext applicationContext;
+
 	public static void main(final String...args) {
-		run(Application.class, args);
+		applicationContext = run(Application.class, args);
+		displayAllBeans();
+	}
+
+	public static void displayAllBeans() {
+		String[] allBeanNames = applicationContext.getBeanDefinitionNames();
+		for(String beanName : allBeanNames) {
+			System.out.println(beanName);
```

Comment:
?

Reply:
removed.. test-code that accidentally made it in...

Reply:
outrageous